smart-contracts/package.json

```diff
@@ -41,6 +41,7 @@
     "lint": "solhint contracts/**/*.sol",
     "ci": "npm run lint && npm run test",
     "build": "truffle compile --reset",
+    "push-dev": "zos push --reset --network development --from 0xe29ec42f0b620b1c9a716f79a02e9dc5a5f5f98a && npm run deploy",
```

Comment:
Hey, not sure this is a good approach because it means we can only push in dev, when we actually need to do that on other occasions, such as CI.

Reply:
I agree. I originally though I could use a variable here, but now I think a cleaner way to do this would be to move this command to a migrations script,
maybe named `2_push_unlock.js`, followed by `3_create_proxy.js`. 
• On any given network, each these scripts will only be run once unless we pass the `--reset` flag to truffle migrate. 
• this keeps zos in migrations and removes it from use in package.json
• we can have fine-grained control over what happens on a network-by-network basis.

Reply:
This is a great approach. Do you want to take that on?

Reply:
I've got it implemented already. At the ski hill with my kids, but will push when home. 2 parts: config and code, so will push config first, then rebase part 2 after part 1 is merged of that sounds good.